jekyll-feed.gemspec

```diff
@@ -19,6 +19,8 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "jekyll", "~> 3.3"
+  spec.required_ruby_version = ">= 2.3.0"
```

Comment:
I recommend this line to be added above `spec.add_dependency "jekyll", "~> 3.3"` so that the plugins runtime dependency and dev-dependencies are grouped close to each other